src/opnsense/service/conf/actions.d/actions_unbound.conf

```diff
@@ -74,6 +74,14 @@ type:script_output
 message:Updating Unbound DNSBLs
 description:Update Unbound DNSBLs
 
+[overrides]
+command:
+    /usr/local/sbin/pluginctl -c hosts &&
```

Comment:
'hosts' also messes with Dnsmasq and we should avoid one way or another

Reply:
would it make sense to split these up on top of the existing `hosts` hook? e.g. `hosts_unbound`. I know it's not particularly clean but I'm missing a simple alternative.